.github/workflows/java-common-release-build-pull-request.yml

```diff
@@ -0,0 +1,201 @@
+name: Release Pull Request
+concurrency:
+  group: release-pr-${{ github.event.pull_request.head.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+    branches: [main]
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - closed
+  workflow_call:
+    inputs:
+      pomVersion:
+        type: string
+        required: false
+        default: ${{ needs.determine-version.outputs.pomVersion }}
+      refFrom:
+        type: string
+        required: false
+        default: ${{ github.event.pull_request.base.sha }}
```

Comment:
Since we are using `workflow_call`, we can't use any variables from `github.event.pull_request`. There are a few different places in this file.